### ISSUE TITLE
make update-asgsh --clean work when asgs was rolled off /work and asgs-brew.pl is not available 

### ISSUE THREAD

**Issue by jasonfleming** [StormSurgeLive/asgs#437]
I set out to completely reinstall asgs on qbc starting with the command `update-asgsh --clean` but it could not because this command relies on `asgs-brew.pl` which was no longer available because the previous asgs installation was installed on `/work` and had been automatically rolled off so `asgs-brew.pl` was not available. 

I will delete the remnants of the previous asgs installation that remain in my home directory, including `~/perl5` and `~/bin/*` but not `asgs-global.conf` or `~/.ssh`. Not sure what to do with `~/.asgs-brew.sh` etc. Here is what I currently have in my home directory:

```
[jgflemin@qbc1 ~]$ ls -a
.   .adcirc-meta  .asgs-brew.sh     .bash_history  .bash_profile  bin     .config  .gitconfig        .kshrc  .modules  perl5        .pki  .subversion  .viminfo
..  .asgs         asgs-global.conf  .bash_logout   .bashrc        .cache  .emacs   .git-credentials  .local  .mozilla  .perltidyrc  .ssh  .thist1
```

Are there any other things that I should be deleting manually, or will they be overwritten by the reinstall process?

**Comment by wwlwpd** [comment]
I was thinking base directory could either be provided via an option or the script to look at `$WORK` as the base directory - moreover this reveals that such a tool probably should be available if there are 2 or more `asgs` installations on the same system, anyway

**Comment by jasonfleming** [comment]
This all works now, most recent changes for isolation were merged with #916
